WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -101,9 +105,40 @@ class BloggingRemindersViewModel @Inject constructor(
         )
     }
 
-    private fun buildEpilogue(): List<BloggingRemindersItem> {
-        // TODO Add epilogue view items
+    @ExperimentalStdlibApi
+    private fun buildEpilogue(bloggingRemindersModel: BloggingRemindersModel?): List<BloggingRemindersItem> {
+        val numberOfDays = bloggingRemindersModel?.enabledDays?.size ?: 0
+
+        val selectedDays = when {
+            numberOfDays <= THREE_DAYS -> {
+                bloggingRemindersModel?.enabledDays?.map { day ->
+                    day.name.toLowerCase(Locale.ROOT).replaceFirstChar {
+                        it.toUpperCase()
+                    }.plus(" and ")
+                }.toString()
+            }
+            numberOfDays in FOUR_DAYS..SIX_DAYS -> {
+                bloggingRemindersModel?.enabledDays?.map { day ->
+                    day.name.toLowerCase(Locale.ROOT).replaceFirstChar {
+                        it.toUpperCase()
+                    }.take(FIRST_THREE_CHARS).plus(", ")
```

Comment:
Same as above, instead of using `take()` here, we should actually use the proper formatting classes to get the localized day of week short name.

I believe we can do this with `DateFormat` although I don't remember the syntax pretty well. That said, when #14698 gets merged, we will be able to do it like so: `DayOfWeek.MONDAY.getDisplayName(TextStyle.SHORT, Locale.getDefault())`.